packages/assets/src/loader/parsers/loadWebFont.ts

```diff
@@ -80,19 +81,26 @@ export const loadWebFont = {
 
                 document.fonts.add(font);
 
-                return font;
+                fontFaces.push(font);
             }
+
+            return fontFaces.length === 1 ? fontFaces[0] : fontFaces;
         }
-        else
-        {
-            console.warn('[loadWebFont] FontFace API is not supported. Skipping loading font');
-        }
+
+        console.warn('[loadWebFont] FontFace API is not supported. Skipping loading font');
 
         return null;
     },
 
-    unload(font: FontFace): void
+    unload(font: FontFace[]): void
     {
-        document.fonts.delete(font);
+        if (Array.isArray(font))
+        {
+            font.forEach((t) => document.fonts.delete(t));
+        }
+        else
+        {
+            document.fonts.delete(font);
+        }
```

Comment:
Slightly more terse way to describe the same thing:
```suggestion
        (Array.isArray(font) ? font : [font])
            .forEach((t) => document.fonts.delete(t));
```